Makefile

```
@@ -0,0 +1,5 @@
+setup:
+	bundle install --path vendor/bundle
```

`--path vendor/bundle` 指定不要です

```suggestion
	bundle install
```